src/test/resources/api/mappings/mapping-contents-fu-nMBRw.json

```diff
@@ -6,13 +6,13 @@
     "method" : "GET"
   },
   "response" : {
-    "status" : 301,
```

Comment:
this looks to have been wrong from the outset and was shown by a fix in the updated client.

checked https://api.github.com/repos/jenkinsci/jenkins/contents/?ref=b9fcd2ae02752e7a74e9ca4886e490e435a1ff03 and a few others to show that they do not show redirects and a body (the header did not even have a `Location` so a `301` would be very unusual and would look like a bug)